app/helpers/wpcc/your_results_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module YourResultsPageHelper
+    TAX_RELIEF_THRESHOLD_RATE = 11_500
+
+    def earning_below_tax_relief_threshold?(salary)
+      salary.to_i < TAX_RELIEF_THRESHOLD_RATE
+    end
+  end
+end
```

Comment:
So I need to read up more on presenters as well; in my past helpers have always been the go to place. But, I agree we should standardise on one pattern or another. I am happy to go with the majority decision on this and would be useful to have a lightning talk perhaps from somebody about the differences etc. 
If @margOnline / @tomas-stefano could pair with @Tr4pSt3R on making this change to help spread the knowledge. I will spend some of my DLT learning about this pattern also as well as reviewing the code produced